todo_app/tests/integration_tests/integration_test.py

```diff
@@ -0,0 +1,113 @@
+from flask import template_rendered
+from pytest import fixture
+from contextlib import contextmanager
+import dotenv
+import json
+
+from todo_app.app import create_app
+from unittest.mock import patch, Mock
+from todo_app.tests.integration_tests.mock_responses import *
+
+@fixture
+def client():
```

Comment:
Given that you actually return the test_app rather than the test_client here, this name isn't very clear